src/components/TextAreaChat/styles.ts

```diff
@@ -1,10 +1,10 @@
 import {StyleSheet} from 'react-native';
 
-import {COLORS} from '../../utils/theme';
+import {colors} from '../../utils/colors';
 
 export const S = StyleSheet.create({
   container: {
-    backgroundColor: COLORS.white,
+    backgroundColor: colors.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement and the color property have been updated correctly. The new color scheme seems to be imported and used properly.
```diff
- import {COLORS} from '../../utils/theme';
- backgroundColor: COLORS.white,
+ import {colors} from '../../utils/colors';
+ backgroundColor: colors.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->